src/components/task-step/step-mixin.cjsx

```diff
@@ -5,10 +5,13 @@ BS = require 'react-bootstrap'
 {TaskStepActions, TaskStepStore} = require '../../flux/task-step'
 LoadableItem = require '../loadable-item'
 {CardBody} = require '../pinned-header-footer-card/sections'
-
+_ = require 'underscore'
 
 module.exports =
 
+  getInitialState: ->
+    classNames: ['task-step']
```

Comment:
Good idea - I think I was going for extendability or something there, but I'm all for doing the simplest thing that would possibly work.  Thanks for pointing that out.